app/src/main/java/io/pslab/fragment/LuxMeterDataFragment.java

```diff
@@ -535,7 +529,7 @@ private void initiateLuxSensor(int type) {
                 if (sensor == null) {
                     Toast.makeText(getContext(), getResources().getString(R.string.no_lux_sensor), Toast.LENGTH_LONG).show();
                 } else {
-                    float max = sensor.getMaximumRange();
+                    float max = sensor.getMaximumRange() * 10000;
```

Comment:
This is the only related change right? Revert everything else back. Too many unnecessary changes moving stuff here and there.

Reply:
I had reformatted the file using [<kbd>CTRL</kbd>+<kbd>ALT</kbd>+<kbd>L</kbd>] thats why all those changes were made. I have reverted them now.